### ISSUE TITLE
`text.com` Becomes `text. Com` With Smart Spaces

### ISSUE THREAD

**Issue by Danc2050** [psoukie/zipchord#127]
**Is your feature request related to a problem? If so, please describe.**
I like the autospace feature after punctuation, but I do not like when typing `.com` it will type ". Com". It adds a space and capitalizes the `com`. 
**Describe the solution or functionality you'd like**
Perhaps a check can be made for TLDs to space back after the space after `com` (so I do not start a new sentence and say "I like you. Come over! 

**Additional context**
Not super high priority I'm sure and I can just get over the small inconvenience or turn off the feature.

**Comment by psoukie** [comment]
Good suggestion! I also noticed similar situations around decimal places with numbers (which will be easier to solve). 

In the meantime, a workaround could be a `.c` chord for `~.com` (to attach as a suffix)

**Comment by Danc2050** [comment]
Neat! I will do that. Thank you for the workaround 🙏

**Comment by psoukie** [comment]
I thought about this more and some of the situations where unwanted smart spaces happen are tricky (and language dependent), so there might not be a good way to address all. (Sure, turning off smart spaces after punctuation is always an option.)

Another, more universal workaround, is to define an “unspaced” full stop (and maybe unspaced comma) in dictionaries:

```
 .	~.~	(spacebar+full stop to full stop that behaves as an infix)
```

I'm going to link two related things to this issue that I can fix and improve, and will keep this open for additional improvements.

**Comment by psoukie** [comment]
I added the contemplated workaround into the default dictionaries as a solution.

(I tested other approaches, but they all have downsides. For example, turning a sequence `.` followed by `com` into `.com` is a possibility, but it backfires in contexts such as "_Text. Commercial applications...` and putting yet another exception on top of that to re-insert the space feels convoluted and appears unpredictable. )

The added entries in the default chord dictionaries are as follows:

```
Infixes
-------
.c	~.com~
.o	~.org~

Punctuation Exceptions
----------------------
These exceptions are used to correct the output when using smart spaces after punctuation.

 .	~.~	Space+. forces a full stop without smart spaces
e . g .	e.g.
i . e . i.e.
```
